kms-message/src/kms_response_parser.c

```diff
@@ -101,6 +111,16 @@ _parse_int_from_view (const char *str, int start, int end, int *result)
    return ret;
 }
 
+static bool
+_parse_hex_from_view (const char *str, int start, int end, int *result)
```

Comment:
Just take length instead of (start, end)

Reply:
Done.